vignettes/dev04_RECIPES.Rmd

```diff
@@ -46,7 +46,7 @@ The `blocks` object is actually an array of `block` objects. The setup may feel
 * The `variable` element denotes the field or parameter from the `data` data set to analyze with a statistical computation.
 * The `statistic` element denotes which statistical procedure to perform on said field/parameter.
 
-A few other elements can also be present within a `block` object, such as: `stat_arg`, `stat_options`, and `stat_options_fn`.  A `stat_arg` element is only required when the `statistic` element requires it. As seen in the example above, the "NESTED_FREQ_ABC" `statistic` needs to know a secondary variable by which to produce its "nested" output. Besides those mentioned, no other elements are used in the `recipes.json` file or within the application's code base, but active development will be changing this soon.
+A few other elements can also be present within a `block` object, such as: `var_selction`, `var_options`, `var_options_fn` and their statistic counterparts `stat_selection`, `stat_options`, and `stat_options_fn`. A `var_selection` element can be used in conjunction with `var_options` to create a subgroup filter for the variable.  A `stat_selection` element is only required when the `statistic` element requires it. As seen in the example above, the "NESTED_FREQ_ABC" `statistic` needs to know a secondary variable by which to produce its "nested" output. Besides those mentioned, no other elements are used in the `recipes.json` file or within the application's code base, but active development will be changing this soon.
```

Comment:
The example above has the old naming convention!

![image](https://user-images.githubusercontent.com/17878953/232888887-89ffc7e2-22ea-43c3-9a96-5d01ed71bb06.png)

It might be easier to put the actual code into the Rmd file instead of relying on a png.

Reply:
It would be nice to see a 2nd example that shows off an `ATPT` example! The second example could be in the same code chunk - just refer to it as table 3 or table x, where "x" is your second example.